recipes/r-rmatio/meta.yaml

```
@@ -0,0 +1,88 @@
+{% set version = '0.16.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rmatio
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rmatio_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rmatio/rmatio_{{ version }}.tar.gz
+  sha256: 73d2d6c4db1cf25535ce3e87a199f700dca338ed0e5d4cfc6484f3d2cd3828ab
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*'
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-matrix
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-matrix
+
+test:
+  commands:
+    - $R -e "library('rmatio')"           # [not win]
+    - "\"%R%\" -e \"library('rmatio')\""  # [win]
+
+about:
+  home: https://github.com/stewid/rmatio
+  license: GPL-2.0-or-later
```

GPL3 from cran meta data resolves to:
```suggestion
  license: GPL-3.0-only
```